Assets/Scripts/Controller/IController.cs

```diff
@@ -5,9 +5,11 @@
 // The controller determines the acceleration input given the agent's current state and its intended
 // target or waypoint.
 public interface IController {
+  IAgent Agent { get; set; }
```

Comment:
Recommend either 1) init-only or 2) constructor-only assignment and remove the setter
```suggestion
  IAgent Agent { get; init; }
```

Reply:
Done in #116